test/sbvr-helper.js

```diff
@@ -0,0 +1,498 @@
+/*
+ * decaffeinate suggestions:
+ * DS104: Avoid inline assignments
+ * DS205: Consider reworking code to avoid use of IIFEs
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+ */
+const _ = require('lodash');
+
+const stripAttributes = (x) => _.reject(x, { 0: 'Attributes' });
+
+const factTypeBody = (fType) =>
+	_(fType)
+		// Standardise the fact type parts
+		.map(function (factTypePart) {
+			if (_.isNumber(factTypePart) || _.isString(factTypePart)) {
+				// Parse numbers/strings to the correct term array.
+				return parseEmbeddedData(factTypePart);
+			} else if (factTypePart[0] === 'Term') {
+				// Strip attributes from terms
+				return stripAttributes(factTypePart);
+			} else if (factTypePart[0] === 'Verb') {
+				return factTypePart;
+			} else {
+				// Ignore any unknown fact type parts
+				return null;
+			}
+		})
+		.compact()
+		.value();
+
+exports.vocabulary = (vocab) => ['Vocabulary', vocab, ['Attributes']];
+exports.term = function (term, vocab) {
+	if (vocab == null) {
+		vocab = 'Default';
+	}
```

Comment:
Yeah, I left it as-is because it's technically different, if/when I add typescript checking I can change it as it should be safe to do then - same reason for esm exports